includes/transformer/class-post.php

```diff
@@ -894,11 +894,15 @@ protected function get_post_content_template() {
 		$template = $content ?: ACTIVITYPUB_CUSTOM_POST_CONTENT; // phpcs:ignore Universal.Operators.DisallowShortTernary.Found
 
 		$post_format_setting = \get_option( 'activitypub_object_type', ACTIVITYPUB_DEFAULT_OBJECT_TYPE );
+		$type                = $this->get_type();
 
 		if ( 'wordpress-post-format' === $post_format_setting ) {
 			$template = '';
 
-			if ( 'Note' === $this->get_type() ) {
+			// If the post is a note and not a reply, force the inclusion of the post title.
+			$is_reply = null !== $this->get_in_reply_to();
+
+			if ( 'Note' === $type && ! $is_reply ) {
```

Comment:
```suggestion
			if ( 'Note' === $type && empty( $this->get_in_reply_to() ) ) {
```
Would this work?

It would be great to have a unit test for this

Reply:
Works for me! c289589abbde6696e917b4ec8d92256c94a18e80